index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
- [ ] Very well done 👏🏻 you did good work on your first project, but please 🙏🏻 adhere to the project requirement according to which you must have to add the class for your header and then by using that class you need to add the style in your `style.css`. I agree that you produce the output but you need to follow the requirement because it will help you in future to work with the given instructions and make you the kind of developer who really pays attention on the small details.

## Requirement

![image](https://github.com/nouman17610/hello-microverse/assets/66476329/6a28658a-dd14-49dd-981b-fb3754bc596d)